### ISSUE TITLE
Release download (windows and others?) missing .zip extension

### ISSUE THREAD

**Issue by nathan-at-mulesoft** [mulesoft-labs/data-weave-native#49]
**Describe the bug**
The manual downloads have downloaded files with no extension. e.g. on windows that come out as dw-X.Y.Z-Windows - should have a .zip at the end. 

**To Reproduce**
Go to the downloads for releases: https://github.com/mulesoft-labs/data-weave-cli/releases
Download the windows one, it'll save as dw-1.0.24-Windows
The source files however have their extensions when downloaded, so they are ok. 

**Expected behavior**
Should have a .zip at the end so people can fire up the unzip tool/know what the format is of the file (on linux/mac it might be ambiguous what the file is, whether it should be executed as is.. on windows people might think they need to add a .exe or something). 

**Desktop:**
 - OS: windows 11
 - Browser: chrome
 - Version: 105.0.5195.127 (Official Build) (64-bit)

**Comment by afelisatti** [comment]
Thanks for bringing this up, @nathan-at-mulesoft. We will take a look. Eventually, we would like to understand the package managers of the different environments and be able to do what we've done with Homebrew/Mac so this isn't a problem at all. If you have any insights around Windows on that it would be helpful.

**Comment by leogitpub** [comment]
Yes. The Windows binary bundle is not a zip. You need to convert that to zip and then extract. But I am also facing issue with the linux binary. It says, "glibc_2.0 not found" while running on Latest version on Mint. Looks like the linux build is linked on an old machine.

**Comment by machaval** [comment]
@omkarium The build is being done on ubuntu latest on the github infrastructure. You can always try to build it your self it should be super simple and validate if that fix the problem. If so we can see what we can do.

**Comment by manikmagar** [comment]
> Yes. The Windows binary bundle is not a zip. You need to convert that to zip and then extract. But I am also facing issue with the linux binary. It says, "glibc_2.0 not found" while running on Latest version on Mint. Looks like the linux build is linked on an old machine.

@omkarium  The WSL on windows seems to be running Ubuntu 18.04, while Ubuntu on GitHub Actions is using Ubuntu 22.04. Maybe that is causing the difference. Try updating WSL to use Ubuntu 22.04. If that works, maybe the CLI can be built on lower Ubuntu's like 20.04.
